gnucash/import-export/import-main-matcher.c

```diff
@@ -553,9 +565,18 @@ on_matcher_ok_clicked (GtkButton *button, GNCImportMainMatcher *info)
         if (first_tran)
         {
             Split* first_split = gnc_import_TransInfo_get_fsplit (trans_info);
-            xaccAccountSetAppendText (xaccSplitGetAccount(first_split), append_text);
+            Account* account = xaccSplitGetAccount (first_split);
+            xaccAccountSetAppendText (account, append_text);
```

Comment:
`xaccAccountSetAppendText` calls `xaccAccountBeginEdit` and `xaccAccountCommitEdit` so might as well be after the `begin_edit` call to stop those being processed until the list destruction.